### PR TITLE
Tidy task page

### PR DESCRIPTION
**Pull request by andysellick**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What
Clean up the page at `/landing-page/task`, specifically:

- change the content in the hero
- correctly configure mid page hero
- adjust the chart block spacing and link output
- add a new block using the document list component

## Why
Doing final checks on the page layout and content testing.

## Visual changes
Some! Hopefully improvements all.

Trello card: https://trello.com/c/t0DlqDkW

### CODE REVIEW

File: app/views/landing_page/blocks/_statistics.html.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 %>
-
 <%= render "govuk_publishing_components/components/chart", {
   chart_heading: block.data["title"],
   h_axis_title: block.data["x_axis_label"],
   v_axis_title: block.data["y_axis_label"],
   hide_legend: block.data["hide_legend"],
   keys: block.x_axis_keys,
   rows: block.rows,
+  link: block.data["data_source_link"],
```
Comment (leenagupte) [inline app/views/landing_page/blocks/_statistics.html.erb:11]: Oh this is much nicer!

File: lib/data/landing_page_content_items/task.yaml [hunk 2]
```diff
@@ -48,7 +48,6 @@ blocks:
       x_axis_label: "X Axis"
       y_axis_label: "Y Axis"
       csv_file: "data_one.csv"
-      data_source_link_text: "Data source"
```
Comment (leenagupte) [inline lib/data/landing_page_content_items/task.yaml:55]: If this label is no longer needed, it should be removed from the other demos as well as the tests and the fixtures.

Perhaps that will be tidied up in a later PR?

File: app/models/block/document_list.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+module Block
+  class DocumentList < Block::Base
+    attr_reader :items
+
+    def initialize(block_hash)
+      super(block_hash)
+
+      @items = data.fetch("items").map do |i|
```
Comment (leenagupte) [inline app/models/block/document_list.rb:8]: Minor, and not a blocker, but as it's doing so much, I'd have made items into a method.

File: spec/models/block/document_list_spec.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+RSpec.describe Block::DocumentList do
+  let(:blocks_hash) do
+    { "type" => "document_list",
+      "items" => [
+        { "text" => "link 1", "path" => "/a-link", "document_type" => "News article", "public_updated_at" => "2024-01-01 10:24:00" },
+        { "text" => "link 2", "path" => "/another-link", "document_type" => "Press release", "public_updated_at" => "2023-01-01 10:24:00" },
+      ] }
+  end
+
+  describe "#items" do
```
Comment (leenagupte) [inline spec/models/block/document_list_spec.rb:10]: Lovely tests! ❤️

**Review by leenagupte — APPROVED** [review]
I've added one minor comment inline, but it's not a blocker.

Since you're looking at the task page, the side navigation is not rendering, but this is something that can addressed in a separate PR.

File: lib/data/landing_page_content_items/task.yaml (hunk 2 above)
Comment (andysellick) [reply lib/data/landing_page_content_items/task.yaml:55]: Good shout, have raised: https://github.com/alphagov/frontend/pull/4313

File: spec/models/block/document_list_spec.rb (hunk 4 above)
Comment (andysellick) [reply spec/models/block/document_list_spec.rb:10]: 🙇

File: app/models/block/document_list.rb (hunk 3 above)
Comment (andysellick) [reply app/models/block/document_list.rb]: Good shout, have converted 👍

**Pull request merged**